examples/accounts/v2/javascript/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
These package.json files should be removed, as the license is covered here:
https://github.com/algorand/docs/blob/staging/LICENSE.md